src/Exception/SnowflakeDbAdapterException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeDbAdapter\Exception;
+
+interface SnowflakeDbAdapterException extends \Throwable
```

Comment:
Tohle není vyjímka ale Interface. Zvolil bych název jako `ExceptionInterface` a podobně.
https://symfony.com/doc/current/contributing/code/standards.html

Reply:
Myslel jsem, že by to bylo zmatečné pak dělat `try {} catch (SnowflakeDbAdapterExceptionInterface $e) {}` někdo by se divil, že se tam chytá interface... ale OK :) 

Každopádně IMHO musí zůstat dlouhý název, aby to bylo jasné, co chytáš. Když bude v use, tak nebude jasné co to je (`catch (ExceptionInterface $e)`)... Čímž, pravda teď trpí BaseException :(

Reply:
Koukal jsem do symfony a tam to maji fakt pojmenovany vzdycky `interface ExceptionInterface`

Reply:
`SnowflakeDbAdapterExceptionInterface` mi prijde "krok zpatky"

Reply:
OK, good enough for Symfony, good enough for us.